docs/upgrading.md

```diff
@@ -16,6 +16,14 @@ $ rake db:migrate
 
 ---
 
+# From v1.0 to v1.3
+
+Fae v1.3 consolidates install migrations. When upgrading from v1.0 to v1.3 <=, Fae will have to be [upgraded to v1.1 and have the new migrations copied](#from-v10-to-v11):
+
+After a successful migration, follow any [additional upgrading steps](#from-v11-to-v12) before proceeding. For more information, please see [the migration consolidation PR](https://github.com/wearefine/fae/pull/84).
+
+---
+
 # From v1.0 to v1.1
```

Comment:
You can delete this section as you removed the migration needed to make this upgrade.